docs/installation.rst

```diff
@@ -26,20 +26,33 @@ In order to run fmriprep in a Docker container, Docker must be `installed
 <https://docs.docker.com/engine/installation/>`_.
 Once Docker is installed, the recommended way to run fmriprep is to use the
 fmriprep-docker_ wrapper, which requires Python and an Internet connection.
+``fmriprep-docker`` is a streamlined command to run fmriprep without having to
+properly mount directories (``fmriprep-docker`` does this for you)
 
 To install::
 
     $ pip install --user --upgrade fmriprep-docker
 
-When run, ``fmriprep-docker`` will generate a Docker command line for you,
-print it out for reporting purposes, and then run the command, e.g.::
+When you run, ``fmriprep-docker`` it will generate a Docker command line for you,
+print it out for reporting purposes, and then run the command without further action
+needed, e.g.::
 
     $ fmriprep-docker /path/to/data/dir /path/to/output/dir participant
     RUNNING: docker run --rm -it -v /path/to/data/dir:/data:ro \
         -v /path/to_output/dir:/out poldracklab/fmriprep:1.0.0 \
         /data /out participant
     ...
 
+For ``fmriprep-docker`` all the options you would typically pass to fmriprep,
```

Comment:
I'm confused by this paragraph. I think it should differentiate between docker arguments that you can pass in (e.g. environment variables and mounting dirs) and the arguments to fmriprep.

Reply:
@oesteban Not sure exactly what you're suggesting here. As in we should add a section for additional `fmriprep-docker`-specific options? For that I would direct readers to [The Docker Wrapper CLI](https://fmriprep.readthedocs.io/en/stable/usage.html#the-docker-wrapper-cli), to avoid cluttering the basic "How to run fMRIPrep" page with those details.